### PR TITLE
Change package naming and split  DemoJourney

### PR DESCRIPTION
**Pull request by dmlemeshko**
## Summary

- `local` package name is confusing, so we change to `branch` to explicitly state simulations are testing Kibana branches.
- DemoJourney is moved to cloud/branch packages for monitoring convenience 

### Checklist

Delete any items that are not applicable to this PR.

- [x] Branch is successfully tested on [Kibana CI](https://kibana-ci.elastic.co/job/elastic+kibana+load-testing/)
- [ ] Unit tests are added

cloud: https://kibana-ci.elastic.co/view/Kibana/job/elastic+kibana+load-testing-cloud/367/
branch: https://kibana-ci.elastic.co/view/Kibana/job/elastic+kibana+load-testing/437/

### CODE REVIEW

File: src/test/resources/log/simulation.log [hunk 1]
```diff
@@ -1,4 +1,4 @@
-RUN	org.kibanaLoadTest.simulation.DemoJourney	demojourney	1606920741968	 	3.4.2
+RUN	org.kibanaLoadTest.simulation.local.DemoJourney	demojourney	1606920741968	 	3.4.2
```
Comment (LeeDr) [inline src/test/resources/log/simulation.log:1]: In this case `local` really means `local` and not `branch`?

Comment (dmlemeshko) [reply src/test/resources/log/simulation.log:1]: this is actually test. I will change it to `branch` for consistency

File: src/test/scala/org/kibanaLoadTest/helpers/LogParser.scala [hunk 2]
```diff
@@ -37,7 +37,7 @@ object LogParser {
     val br = new BufferedReader(new InputStreamReader(fsStream))
     val strLine = br.readLine
     fsStream.close()
-    // [RUN	org.kibanaLoadTest.simulation.DemoJourney	demojourney	1605016152595	 	3.3.1]
+    // [RUN	org.kibanaLoadTest.simulation.local.DemoJourney	demojourney	1605016152595	 	3.3.1]
```
[the remark this answers is not in the document]
Comment (dmlemeshko) [reply src/test/scala/org/kibanaLoadTest/helpers/LogParser.scala:40]: Another test, will do the same. Thank you

[the remark this answers is not in the document]
Comment (dmlemeshko) [reply src/test/scala/org/kibanaLoadTest/helpers/LogParser.scala:40]: Ah, not a test. Just a side not about skipping the first line

**Pull request merged**